tests/all/volumes.t

```diff
@@ -152,13 +152,33 @@ nodes.each do |node|
   TEST "systemctl restart kadalu-mgr"
   puts TEST "ps aux | grep 'glusterfsd'"
   EQUAL "1", (TEST "ps aux | grep '[g]lusterfsd'| wc -l").strip, "Check for equal number of services[brick-processes]"
-  puts TEST "cat /var/log/kadalu/storage_units/*;"
 end
 
 USE_NODE nodes[0]
 TEST "kadalu volume stop DEV/vol14 --mode=script"
 TEST "kadalu volume delete DEV/vol14 --mode=script"
 
+# Volfile Server tests
+# Volume create with known Storage unit port
+TEST "mkdir -p /exports/vol_volfile/s1"
+TEST "kadalu volume create DEV/vol_volfile server1:5007:/exports/vol_volfile/s1"
+puts TEST "ls /var/lib/kadalu/volfiles"
+
+# Mount the Volume
+TEST "mkdir -p /mnt/vol_volfile"
+TEST "glusterfs -s server1:5007 --volfile-id client-DEV-vol_volfile -l/tmp/volspec.log /mnt/vol_volfile"
+puts TEST "df /mnt/vol_volfile"
```

Comment:
all this worked? Thats good. 

I would prefer to merge the latest changes from glusterfs to this branch before release. kadalu_1 may undergo a hard refresh this week. Lets run a test once that happens too.

Reply:
This is working. I can see updated `log-level` in the client log.

Reply:
Log from CI

```
# [2022-09-07 06:06:17.502048 +0000] I [MSGID: 100030] [glusterfsd.c:2862:main] 0-glusterfs: Started running version [{arg=glusterfs}, {version=2022.09.07}, {cmdlinestr=glusterfs -s server1:5007 --volfile-id client-DEV-vol_volfile -l/tmp/volspec.log /mnt/vol_volfile}] 
# [2022-09-07 06:06:17.506426 +0000] I [MSGID: 0] [glusterfsd.c:2552:daemonize] 0-glusterfs: Pid of current running process is 2283 
# [2022-09-07 06:06:17.512175 +0000] I [MSGID: 101188] [event-epoll.c:660:event_dispatch_epoll_worker] 0-epoll: Started thread with index [{index=0}] 
# [2022-09-07 06:06:17.512237 +0000] I [MSGID: 101188] [event-epoll.c:660:event_dispatch_epoll_worker] 0-epoll: Started thread with index [{index=1}] 
# [2022-09-07 06:06:17.514172 +0000] I [MSGID: 0] [io-stats.c:3797:ios_sample_buf_size_configure] 0-vol_volfile: Configure ios_sample_buf  size is 1024 because ios_sample_interval is 0 
# [2022-09-07 06:06:17.514435 +0000] I [MSGID: 114020] [client.c:2344:notify] 0-vol_volfile-distribute-0-client-0: parent translators are ready, attempting connect on transport [] 
# Final graph:
# +------------------------------------------------------------------------------+
#   1: volume vol_volfile-distribute-0-client-0
#   2:     type protocol/client
#   3:     option transport-type socket
#   4:     option remote-port 5007
#   5:     option remote-host server1
#   6:     option remote-subvolume /exports/vol_volfile/s1
#   7: end-volume
#   8:  
#   9: volume vol_volfile-distribute-0
#  10:     type cluster/distribute
#  11:     subvolumes vol_volfile-distribute-0-client-0
#  12: end-volume
#  13:  
#  14: volume vol_volfile-utime
#  15:     type features/utime
#  16:     subvolumes vol_volfile-distribute-0
#  17: end-volume
#  18:  
#  19: volume vol_volfile-write-behind
#  20:     type performance/write-behind
# [2022-09-07 06:06:17.514829 +0000] I [MSGID: 114057] [client-handshake.c:871:select_server_supported_programs] 0-vol_volfile-distribute-0-client-0: Using Program [{Program-name=GlusterFS 4.x v1}, {Num=1298437}, {Version=400}] 
#  21:     subvolumes vol_volfile-utime
#  22: end-volume
#  23:  
#  24: volume vol_volfile
#  25:     type debug/io-stats
#  26:     subvolumes vol_volfile-write-behind
#  27: end-volume
#  28:  
#  29: volume meta-autoload
#  30:     type meta
#  31:     subvolumes vol_volfile
#  32: end-volume
#  33:  
#  34: volume meta-autoload
#  35:     type meta
#  36:     subvolumes meta-autoload
#  37: end-volume
#  38:  
# +------------------------------------------------------------------------------+
# [2022-09-07 06:06:17.520007 +0000] I [MSGID: 114046] [client-handshake.c:621:client_setvolume_cbk] 0-vol_volfile-distribute-0-client-0: Connected, attached to remote volume [{conn-name=vol_volfile-distribute-0-client-0}, {remote_subvol=/exports/vol_volfile/s1}] 
# [2022-09-07 06:06:17.521599 +0000] I [MSGID: 0] [fuse-bridge.c:5328:fuse_init] 0-glusterfs-fuse: FUSE inited with protocol versions: glusterfs 7.24 kernel 7.34 
# [2022-09-07 06:06:17.521621 +0000] I [MSGID: 0] [fuse-bridge.c:5960:fuse_graph_sync] 0-fuse: switched to graph 0 
# [2022-09-07 06:06:17.754070 +0000] I [MSGID: 0] [glusterfsd-mgmt.c:35:mgmt_cbk_spec] 0-mgmt: Volume file changed 
# [2022-09-07 06:06:17.754691 +0000] I [MSGID: 0] [rpc-clnt.c:1878:rpc_clnt_reconfig] 0-vol_volfile-distribute-0-client-0: changing ping timeout to 42 (from 0) 
# [2022-09-07 06:06:17.754776 +0000] I [MSGID: 0] [io-stats.c:3797:ios_sample_buf_size_configure] 0-vol_volfile: Configure ios_sample_buf  size is 1024 because ios_sample_interval is 0 
# [2022-09-07 06:06:17.754789 +0000] I [MSGID: 0] [options.c:1371:xlator_option_reconf_str] 0-vol_volfile: option log-level using set value DEBUG 
# [2022-09-07 06:06:17.754802 +0000] D [MSGID: 0] [io-stats.c:3939:reconfigure] 0-vol_volfile: reconfigure returning 0 
# [2022-09-07 06:06:17.754808 +0000] D [MSGID: 0] [options.c:1242:xlator_reconfigure_rec] 0-vol_volfile: reconfigured 
# [2022-09-07 06:06:17.754813 +0000] D [MSGID: 0] [options.c:1262:xlator_reconfigure_rec] 0-meta-autoload: No reconfigure() found 
# [2022-09-07 06:06:17.754850 +0000] D [MSGID: 0] [glusterfsd-mgmt.c:2423:mgmt_getspec_cbk] 0-glusterfsd-mgmt: No need to re-load volfile, reconfigure done 
```